Packs/CortexCloudSec/Integrations/AWS/AWS_test.py

```diff
@@ -0,0 +1,192 @@
+import importlib
+import pytest
+
+AWS_INTEGRATION = importlib.import_module("AWS")
+
+class AWSClient:
+    def aws_session(self, **kwargs):
+        return Boto3Client()
+
+class Boto3Client:
+    def get_public_access_block(self, **kwargs):
+        pass
+
+    def put_public_access_block(self, **kwargs):
+        pass
+
+    def get_account_password_policy(self, **kwargs):
+        pass
+
+    def update_account_password_policy(self, **kwargs):
+        pass
+    
+    def modify_instance_metadata_options(self, **kwargs):
+        pass
+
+AWS_CLIENT = AWSClient()
```

Comment:
Lets avoid using globals and keep the unit tests atomic.
Instead define a fixture that returns an instance of `AWSClient`, and use it for all the tests.
```suggestion
 @pytest.fixture()
 def aws_client():
    return AWSClient()
```